mypy.ini

```diff
@@ -8,7 +8,14 @@ exclude = (?x)(
           | baybe/searchspace
           | baybe/strategies
           | baybe/surrogates
-          | baybe/utils
```

Comment:
Is there any specific reason for this chance other than making it more explicit and easier to have more atomic PRs in the future?

Reply:
I think you missed the actual important change: I wanted to include `interval.py`, that's why I needed to remove the folder ignore and replaced it with file ignores for all contained files **except** `interval.py`